src/utils/allowedHTML.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Defines the list of allowed HTML tags that do not count towards the total
+ * character limit.
+ *
+ */
+const regrexTests = [
+  new RegExp("(<iframe.*/iframe>)", "gm"),
+  new RegExp("(<div.*/div>)", "gm"),
+]
```

Comment:
Hmm why do we also allow div tags to bypass the check? This feels like it might be easily abusable to get abnormally long lines

Reply:
Also nit: `regexTests`!

Reply:
hmm I only added it since our users used a `div` to make align their `iframe` - but I agree that this is easily abusable, made the change likewise